Sources/FaunaDB/HighPrecisionTime.swift

```diff
@@ -26,6 +35,8 @@ public struct HighPrecisionTime {
         self.nanosecondsOffset = (adjust % limit) * factor
     }
 
+    /// Converts to a `Foundation.Date`.
+    /// - Note: Ignores the nanoseconds precision.
```

Comment:
I don't really have a problem with this as written, but usually this sort of thing is written as:

"Truncates the nanosecond component", "Drops the nanosecond component", or even "the nanosecond component will be lost." 

"ignore" could be construed as saying that the nanosecond component is still stored, but just not used, which is not the case here